src/app/app.component.ts

```diff
@@ -78,4 +85,15 @@ export class AppComponent {
        timer(3000).subscribe(() => {  this.showSplash = false;});
     });
   }
+
+  rollout() {
+    this.authService.signOut((error, data) => {
+      if (error) {
+        console.log('Login Page : Sign Out Failure');
+      } else {
+        console.log('Login Page : Sign Out Success');
+      }
+      this.router.navigateByUrl('/login');
```

Comment:
Either way, go to login page. I think that's fine for this case although there may be good reason to handle failures here.